src/value_generators/test/DocumentGeneratorTestCases.yml

```diff
@@ -177,6 +183,19 @@ Tests:
           xYYXCh/Ybl: { "$numberLong": "9" }
           EHcmXZX/d4: { "$numberLong": "10" }
 
+  - Name: Object Generator disallow duplicate keys
```

Comment:
I'm not sure how to run the tests from this file. From the instructions in https://github.com/mongodb/genny/blob/b3e5f1775a65869fd4413857cbd19638e0e4b33f/docs/developing.md it's not obvious to me which kind of tests this one is. There is also no `src/resmokeconfig/genny_standalone.yml` file and when I tried `./run-genny resmoke-test --create-new-actor-test-suite` it failed with `No such file or directory: '../../src/genny/build/src/cast_core/cast_core_test'` message.

Reply:
Ya this is a known issue. [This jira comment](https://jira.mongodb.org/browse/EVG-18189?focusedCommentId=4971961&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-4971961) should help.